### PR TITLE
dockerfile updated

### PR DESCRIPTION
**Pull request by MeerKatDev**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,28 +1,46 @@
...
+  mkdir /working/ && \
+  cd /working/ && \
+  curl -L -o sbt-$SBT_VERSION.deb https://dl.bintray.com/sbt/debian/sbt-$SBT_VERSION.deb && \
+  dpkg -i sbt-$SBT_VERSION.deb && \
+  rm sbt-$SBT_VERSION.deb && \
+  apt-get update && \
+  apt-get install sbt && \
+  cd && \
+  rm -r /working/ && \
+  sbt sbtVersion
```
Comment (ErikSchierboom) [inline Dockerfile:21]: Why not directly use the `mozilla/sbt` image?

```dockerfile
FROM mozilla/sbt:8u232_1.4.5
```

File: Dockerfile [hunk 2]
```diff
@@ -1,28 +1,46 @@
...
 
-COPY --chown=gradle:gradle . .
+# COPY --chown=gradle:gradle . .
 #RUN sbt -Dsbt.ivy.home=.ivy2 -Dsbt.global.base=.sbt/1.0 -Dsbt.boot.directory=.sbt/boot test
 RUN mkdir -p /opt/scala-test-runner
 RUN mkdir -p /opt/scala-test-runner/lib
 
 #RUN cp /home/gradle/target/scala-2.12/*.jar /opt/scala-test-runner/lib
 COPY bin/run.sh /opt/scala-test-runner/bin/
+# RUN sbt compile
```
Comment (ErikSchierboom) [inline Dockerfile:35]: This will need to be uncommented at some point at this compiles the actual test runner.

**Review by ErikSchierboom — APPROVED** [review]
This now build! First step towards getting a working Docker image.

File: Dockerfile (hunk 1 above)
Comment (MeerKatDev) [reply Dockerfile]: tried to address both points, it should work now even better now :)

**Comment by ErikSchierboom** [comment]
@MeerKatDev This now is almost fully functional. There are three remaining TODOs that I'll create issues for.

**Pull request merged**